src/Maestro/maestro-angular/src/app/page/channel-graph/channel-graph.component.html

```diff
@@ -0,0 +1,5 @@
+<div style="text-align: center;">
+    <svg stroke="currentColor" class="flowgraph" width=95% height=95% viewBox="0 0 1200 1200" preserveAspectRatio="xMinYMid meet"></svg>
```

Comment:
>width=95% height=95% [](start = 49, length = 20)

What are these "width" and "height" that aren't quoted?  Are these, like, HTML 1.0 height/width properties?  I believe these should just be height: and width: values on the .flowgraph class.

Reply:
these are actually svg attributes, they should probably be quoted though. I don't think they can be in the css.

Reply:
It seemed to work putting them in the css